locales/pt.json

```diff
@@ -31,11 +31,11 @@
  	"use testrpc as the rpc simulator [%s]": "usa testrpc como simulator de rpc [%s]",
  	"port to run the rpc simulator (default: %s)": "porta para executar simulador de rpc (predefinido: %s)",
  	"host to run the rpc simulator (default: %s)": "host para executar servidor de rpc (predefinido: %s)",
- 	"number of accounts (default: %s)": "numero de contas (predefinido: %s)",
- 	"Amount of ether to assign each test account (default: %s)": "Quantidade de éter para atribuir cada conta de teste (predefinido: %s)",
+ 	"number of accounts (default: %s)": "número de contas (predefinido: %s)",
+ 	"Amount of ether to assign each test account (default: %s)": "Quantidade de ether para atribuir cada conta de teste (predefinido: %s)",
  	"custom gas limit (default: %s)": "limite de gás (predefinido: %s)",
  	"run tests": "executar os testes",
- 	"resets embarks state on this dapp including clearing cache": "recomenca o estado do Embark nesta appliacao, incluindo a cache",
+ 	"resets embarks state on this dapp including clearing cache": "recomeçando o estado do Embark nesta appliacao, incluindo a cache",
```

Comment:
`recomeçando` here is wrong. We want to convey "resetting", not "restarting."

Reply:
Which one do you think would fit better? ```resetando``` or ```redefinindo```?